src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -94,24 +120,73 @@ void CommissionerDiscoveryController::OnUserDirectedCommissioningRequest(UDCClie
     mReady = false;
     Platform::CopyString(mCurrentInstance, state.GetInstanceName());
     mPendingConsent = true;
-    char rotatingDeviceIdHexBuffer[RotatingDeviceId::kHexMaxLength];
-    Encoding::BytesToUppercaseHexString(state.GetRotatingId(), state.GetRotatingIdLength(), rotatingDeviceIdHexBuffer,
-                                        RotatingDeviceId::kHexMaxLength);
+    char rotatingIdString[Dnssd::kMaxRotatingIdLen * 2 + 1];
```

Comment:
So is there a reason this is no longer using `RotatingDeviceId::kHexMaxLength`?  If there is, there should probably some documentation where that constant is defined explaining what it means, because on its face it seems like it would be usable here.